yb-voyager/src/query/queryparser/ddl_processor.go

```diff
@@ -122,7 +122,28 @@ func (tableProcessor *TableProcessor) Process(parseTree *pg_query.ParseResult) (
 	}
 
 	// Parse columns and their properties
-	for _, element := range createTableNode.CreateStmt.TableElts {
+	tableProcessor.parseTableElts(createTableNode.CreateStmt.TableElts, table)
+
+	if table.IsPartitioned {
+
+		partitionElements := createTableNode.CreateStmt.GetPartspec().GetPartParams()
+		table.PartitionStrategy = createTableNode.CreateStmt.GetPartspec().GetStrategy()
```

Comment:
consider using protomsg approach here

Reply:
This is the old code, so I haven't changed it. I am just using the already-written code in this case.